file_browser/src/file_browser/local/partials/1-elements/_icons.scss

```diff
@@ -0,0 +1,11 @@
+.icon {
+    width: 25px;
+    height: auto;
+  }
+  
+  .icon-button {
```

Comment:
After updating the generic `*=button` selector, you'll need to override more properties here.